src/renderer/components/FtSearchFilters/FtSearchFilters.vue

```diff
@@ -50,6 +50,12 @@
       />
     </FtFlexBox>
     <div class="searchFilterCloseButtonContainer">
+      <FtButton
+        :label="$t('Clear filters')"
```

Comment:
Also need an entry in `static/locales/en-US.yaml` so people can translate
```suggestion
        :label="$t('Search Filters.Clear Filters')"
```

Reply:
Thanks for the quick response, have committed those changes